windows/RNFS/RNFSManager.h

```diff
@@ -0,0 +1,180 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#pragma once
+#include "NativeModules.h"
+#include <string>
+#include <mutex>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.Security.Cryptography.h>
+#include <winrt/Windows.Security.Cryptography.Core.h>
+#include <winrt/Windows.Web.Http.h>
+
+namespace Cryptography = winrt::Windows::Security::Cryptography;
+namespace CryptographyCore = winrt::Windows::Security::Cryptography::Core;
+namespace RN = winrt::Microsoft::ReactNative;
+
+struct CancellationDisposable
```

Comment:
make this and other structures/classes `final` whenever possible

Reply:
Added the final keyword here.